osgbuild/main.py

```diff
@@ -357,13 +357,14 @@ def parse_cmdline_args(argv):
         dest="redhat_release",
         type="string",
         help="The version of the distribution to build the package for. "
-        "Valid values are: 5 (for RHEL 5), 6 (for RHEL 6), 7 (for RHEL 7). "
-        "If not specified, will build for all releases (koji task) or the "
-        "platform you are running this on (other tasks)")
+        "Valid values are: 5/6/7 (for EL 5/6/7 respectively). "
+        "Default: build for all releases (koji task) current platform "
+        "(other tasks)")
     parser.add_option(
```

Comment:
This confuses me. The original text was better, but if you really want to work in the word “default”, maybe something like: “The default for a koji task is to build for all releases, and for other tasks is to build on the same platform as the command is run on.”